sdk/resources/mgmt/src/test/java/com/azure/resourcemanager/resources/core/AuthFile.java

```diff
@@ -1,10 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
+// Licensed under the MIT License. See License.txt in the project root for license information.
 
 package com.azure.resourcemanager.resources.core;
 
 import com.azure.core.credential.TokenCredential;
-import com.azure.core.implementation.TypeUtil;
```

Comment:
@ChenTanyi 

Please try find time to remove dependency on implementation package.